src/v1/components/surfaces/FloatingPanel/DraggableFloatingPanel.tsx

```diff
@@ -50,28 +48,29 @@ interface DraggableFloatingPanelProps extends HTMLAttributes<HTMLDivElement> {
 
 const DraggableFloatingPanel: React.FC<DraggableFloatingPanelProps> = ({
   bounds,
-  count = 0,
-  icon,
   defaultPosition = { x: 50, y: 150 },
   dragHandleClassName = "drag-handle",
   targetId,
-  title,
   children,
   ...props
 }) => {
-  const theme = useTheme();
   const context = useFloatingPanelContext();
+  const theme = useTheme()
 
-  const backgroundColor =
-    theme.palette.mode === "dark"
-      ? theme.palette.grey[900]
-      : theme.palette.grey[300];
-
-  const visible =
-    (!context.panels[targetId]?.minimised &&
-      context.panels[targetId]?.visible) ||
+  const visible = (!context.panels[targetId]?.minimised &&
+    context.panels[targetId]?.visible) ||
     false;
```

Comment:
idea:
might as well get rid of the false. Feels like artifact of debugging/earlier code